server/tests/test_db.py

```diff
@@ -532,62 +532,6 @@ async def test_migration_v5(self):
     async def test_migration_v6(self):
         await migrations.execute_migrations(self.c, self.dbconn.config, 5)
 
-        with patch('uuid.uuid4') as mock_uuid:
```

Comment:
TLDR: Migration tests should not cover anything else than migrations, no other `connector`-related functions, since these are not versioned in the same way as migrations.

These tests had to go, since they rely on other functions, which have no concept of migration and do not work now (since at v6 there is no `is_banned` column) - this would be a problem with any other modification to `users` table.